aw_watcher_afk/afk.py

```diff
@@ -14,193 +14,146 @@
 
 if platform.system() == "Windows":
     from .windows import time_since_last_input as _time_since_last_input_winfail
+elif platform.system() in ["Darwin", "Linux"]:
+    from .unix import time_since_last_input as _time_since_last_input_unix
 
-settings = {}
 
-def _time_since_last_input_unix():
-    raise NotImplementedError
+class Settings:
+    def __init__(self, config_section):
+        self.timeout = config_section.getfloat("timeout")
+        self.update_interval = config_section.getfloat("update_interval")
+        self.check_interval = config_section.getfloat("check_interval")
+        # TODO: This is a better name for whichever variable above is this one
+        self.polling_interval = 1
 
 
-def time_since_last_input():
+def get_time_since_last_input():
     system = platform.system()
-    if system == "Darwin" or system == "Linux":
+    if system in ["Darwin", "Linux"]:
         return _time_since_last_input_unix()
     elif system == "Windows":
         return _time_since_last_input_winfail()
     else:
         raise Exception("unknown platform")
 
 
-def main() -> None:
-    """ Set up argparse """
-    parser = argparse.ArgumentParser("A watcher for keyboard and mouse input to detect AFK state")
-    parser.add_argument("-v", dest='verbose', action="store_true",
-                        help='run with verbose logging')
-    parser.add_argument("--testing", action="store_true",
-                        help='run in testing mode')
-    parser.add_argument("--desktop-notify", action="store_true",
-                        help='sends desktop notifications when you become afk/non-afk')
-    args = parser.parse_args()
+class AFKWatcher:
```

Comment:
Now when we have a class for it, wouldn't it also make sense to move it to its own file aswell?